string source_cam
string target_cam
uint32 id
---
bool res
